docker/ci/dockerfiles/ci-runner.docker-builder.ubuntu2004.x64.dockerfile

```diff
@@ -17,7 +17,17 @@ FROM ubuntu:20.04
 RUN apt-get update -y && apt-get install -y software-properties-common && add-apt-repository ppa:jacob/virtualisation -y
 
 # Install necessary packages
-RUN apt-get update -y && apt-get upgrade -y && apt-get install -y binfmt-support qemu qemu-user qemu-user-static docker.io curl && apt clean -y
+RUN apt-get update -y && apt-get upgrade -y && apt-get install -y binfmt-support qemu qemu-user qemu-user-static docker.io curl python3-pip && apt clean -y && pip3 install awscli==1.22.12
+
+# Install JDK
+RUN curl -SL https://github.com/AdoptOpenJDK/openjdk14-binaries/releases/download/jdk-14.0.2%2B12/OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.tar.gz -o /opt/jdk14.tar.gz && \
```

Comment:
we didn't need this to build opensearch and opensearch dashboards docker images?

Reply:
Like I said in the description it is for data-prepper.

Reply:
I think I didnt ask it properly. Let me try again:

How were we able to build opensearch and opensearch-dashboards images without jdk

Reply:
Because in opensearch and dashboards we use shell scripts.
Data prepper is using gradle to build.

Reply:
I see! Thanks!